docs/.pages

```diff
@@ -1,7 +1,8 @@
 arrange:
     - index.md
-    - INSTALL.md
+    - install.md
```

Comment:
I based this decision on the list of things requested by GitHub's Community feature (see [here](https://github.com/Materials-Consortia/optimade-python-tools/community)).
If you wish more uniformity that's fine. I'd speak for moving more and more stuff into `/docs` instead of the root of the repo.

Reply:
No it makes sense, we are guided by GH after all...